sample/trace_key.go

```diff
@@ -42,16 +43,29 @@ func (d *traceKey) build(trace *types.Trace) string {
 	// if they happen multiple times.
 	fieldCollector := make(map[string][]string)
 
+	// once a key gets this many unique values, it's off the charts in terms of uniqueness
+	// so we just stop looking for more.
+	// This is a safety valve to prevent us from someone sending a high-cardinality field
+	// in a giant trace.
+	const maxKeyLength = 100
```

Comment:
Probably for the future if the need arises, but this might be a nice configuration option in the future.

Reply:
Created a follow-up issue to track adding in the future if we decide to do it.